packages/main/src/Option.hbs

```diff
@@ -0,0 +1,13 @@
+{{>include "./ListItemBase.hbs"}}
+
+{{#*inline "listItemContent"}}
+	<div part="content" id="{{_id}}-content" class="ui5-li-content">
+		{{#if displayIconBegin}}
+			<ui5-icon part="icon" name="{{icon}}" class="ui5-li-icon" mode="Decorative"></ui5-icon>
+		{{/if}}
+		<span part="title" class="ui5-li-title"><slot></slot></span>
+		{{#if additionalText}}
+			<span part="additional-text" class="ui5-li-additional-text">{{additionalText}}</span>
```

Comment:
`ui5-li-additional-text` doesn't add styles since it's firstly declared in ListItem.css

![image](https://github.com/SAP/ui5-webcomponents/assets/31909318/ab223118-2eca-4e65-8159-0c3a3598ae68)

Reply:
Done. Some content related styles are moved to ListItemBase.css (.ui5-li-content, .ui5-li-text-wrapper)